src/Divider/Divider.js

```diff
@@ -22,7 +23,12 @@ export const styleSheet = createStyleSheet('Divider', (theme) => {
   };
 });
 
-export default function Divider(props, context) {
+type Props = {
+  absolute?: boolean,
+  className?: string,
+};
+
+export default function Divider(props:Props, context:{styleManager: Object}):Element {
```

Comment:
I think I tried it early on, and it didn't work so I yanked it.  It's worth a shot again, would have to run before `flow-vars` because that strips all flow for the remainder of the linting.